.rubocop.yml

```diff
@@ -8,3 +8,7 @@ Metrics/BlockLength:
   Exclude:
     - 'spec/**/*.rb'
     - 'app/admin/*.rb'
+
+Metrics/MethodLength:
```

Comment:
Dont' do this, dont exclude a method length in all of the controllers. Which method gives you problem?

Reply:
The create method of rents_controller.rb. What should i do with rubocop's concerns about the method's length?

Reply:
Try to fix it so you don't need to disable the method length in ALL THE CONTROLLERS, maybe adding some new spaces for example